docs/cpp/config-clang-mac.md

```diff
@@ -131,155 +150,189 @@ Next, we need to create a `tasks.json` file to tell VS Code how to build (compil
 
-   ![Clang build output in terminal](images/clang-mac/clang-task-in-terminal.png)
+1. Press **Step over** again to execute the `cout` statement. **Note** As of the March 2019 version of the extension, no output will appear in the DEBUG CONSOLE until the last `cout` completes.
 
-1. As the message instructs, press any key to close the integrated terminal.
+## Set a watch
 
-## Start a debugging session
+Sometimes you might want to keep track of the value of a variable as your program executes. You can do this by setting a **watch** on the variable.
 
-> **Note**: You may have issues debugging on macOS Catalina. See [issue #3829](https://github.com/microsoft/vscode-cpptools/issues/3829) for details and possible workarounds.
+1. Place the insertion point inside the loop. In the **Watch** window, click the plus sign and in the text box, type `word`, which is the name of the loop variable. Now view the **Watch** window as you step through the loop.
 
-1. You are now ready to run the program. Press `kb(workbench.action.debug.start)` or from the main menu choose **Run > Start Debugging**. Before we start stepping through the code, let's take a moment to notice several changes in the user interface:
+   ![Watch window](images/cpp/watch-window.png)
 
-- The **Debug Console** appears and displays output from the debugger.
+1. To quickly view the value of any variable while execution is paused, you can hover over it with the mouse pointer.
 
-- The code editor highlights the first statement in the `main` method. This is a breakpoint that the C++ extension automatically sets for you:
+   ![Mouse hover](images/cpp/mouse-hover.png)
 
-![Initial breakpoint](images/cpp/breakpoint-default.png)
+## C/C++ configurations
 
-- The workspace pane on the left now shows debugging information. These windows will dynamically update as you step through the code.
+If you want more control over the C/C++ extension, you can create a `c_cpp_properties.json` file, which will allow you to change settings such as the path to the compiler, include paths, C++ standard such as C++17), and more.
 
-![Debugging windows](images/clang-mac/mac-debugging-windows.png)
+You can view the C/C++ configuration UI by running the command **C/C++: Edit Configurations (UI)** from the Command Palette (`kb(workbench.action.showCommands)`).
 
-- At the top of the code editor, a debugging control panel appears. You can move this around the screen by grabbing the dots on the left side.
+![Command Palette](images/clang-mac/mac-command-palette-configurations.png)
 
-![Debugging controls](images/cpp/debug-controls.png)
+This opens the **C/C++ Configurations** page. When you make changes here, VS Code writes them to a file called `c_cpp_properties.json` in the `.vscode` folder.
 
-## Step through the code
+![C++ configuration](images/msvc/configurations-ui.png)
```

Comment:
Still pointing to the MSVC image. I think you created a new one for macOS.

Reply:
Thank you - would have hated to build the screenshot for nothing. Fixed.